.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Denote all files that are truly binary and should not be modified.
+tests/data/bpftrace/* binary
```

Comment:
While we're at it, maybe denote this for the other binary files too?

Reply:
👍